topics/admin/tutorials/tool-management/tutorial.md

```diff
@@ -77,7 +77,7 @@ Let's try running this on a real worfklow.
 >
 > 1. Download the mapping workflow:
 >    ```console
->    $ wget {{ site.url }}/topics/sequence-analysis/tutorials/mapping/workflows/mapping.ga
+>    $ wget {{ site.url }}{{ site.baseurl }}/topics/sequence-analysis/tutorials/mapping/workflows/mapping.ga
```

Comment:
That looks suspicious ...

Reply:
It is correct. URL is just protocol + fqdn. Baseurl is only the /... part.